src/components/NurseryWithdrawals/PlantingProgressList.tsx

```diff
@@ -112,7 +114,20 @@ export default function PlantingProgressList({
     return <CircularProgress sx={{ margin: 'auto' }} />;
   }
 
-  const setPlantingCompleted = (complete: boolean) => {
+  const onModalSubmit = () => {
+    setShowWarningModal(false);
+    setPlantingCompleted(false, true);
+  };
+
+  const setPlantingCompleted = (complete: boolean, skipModal?: boolean) => {
+    if (!complete && !skipModal) {
+      const haveStatistics = true;
```

Comment:
@karthikbtf @charlie-tf I'm missing how to set if the subzone has statistics calculated

Reply:
Good question. When a planting subzone is marked completed, the BE calculates statistics like planting density etc based on observation results that were completed on or before the plantingCompletedTime.
How do we get this info now? I think we may need to look at the api and selectors we already have perhaps. Will defer to Charlie here since we need this for the map as well.

Reply:
getting this in first and patching up with the checking code later is fine by me

Reply:
I think that "statistics calculated" means that all of the subzones for a zone have been marked as complete, thereby triggering the BE to calculate aggregate statistics for the zone. Will have to think about how to detect this...

Reply:
```
    ObservationPlantingZoneResultsPayload: {
      /**
       * Format: int32
       * @description Estimated number of plants in planting zone based on estimated planting density and planting zone area. Only present if all the subzones in the zone have been marked as having completed planting.
       */
      estimatedPlants?: number;
```

Reply:
And somehow factor in the plantingCompletedTime as well (I guess skip observation results completed after this time). Ideally there should be only one observation round in motion .. most of my corner cases probably only impact testing.

Reply:
Here's selectLatestObservation: https://github.com/terraware/terraware-web/blob/main/src/redux/features/observations/observationsSelectors.ts#L106

Reply:
Cool, that would work. 
I would suggest a new selector that takes in a list of zone-ids and checks for this (since list can do batch undos). 
The selector can be shared by list and map components. thoughts?
cc @constanzauanini maybe that clarifies?

Reply:
>a new selector that takes in a list of zone-ids and checks for this (since list can do batch undos).
The selector can be shared by list and map components. thoughts?

That sounds good!

Reply:
Thanks @charlie-tf 🙏

Reply:
Thanks @charlie-tf and @karthikbtf! Yes, this clarifies. I will create the selector to check for this. I will get this in first, and open a follow up pr